macros/models/_sector/dex/dex_fork_trades.sql

```diff
@@ -0,0 +1,108 @@
+{% macro dex_fork_v2_trades(
```

Comment:
I think this marco is applicable only for uniswap (or their forks) as you rely on `Pair_evt_Swap` schema and `Factory_evt_PairCreated` table. Check `openocean_optimism_trades` for example.

Reply:
yeah, most likely all uniswap forks, but we have a ton in the dex lineage. it felt weird naming the macro `uniswap_fork_` instead of something generic. happy to revisit though, if others feel like it's more important to call out that these contracts were forked from uniswap. thought maybe leaving out project specific names would be better

Reply:
My vote is for "uniswap_fork..."

Reply:
@Hosuke @0xRobin @hildobby @aalan3 
anyone have an opinion on naming standard for dex macros which are more than likely uniswap forks (i don't know how to quickly validate that)? a generic name like above, or just `uniswap_fork_<version>_trades`?

in this PR, we have one example dex fork, but there'll be plenty more to come

Reply:
+1 for making it clear in the macro name what the original project being forked is. Also, anticipating there'll be multiple projects to be forked, wouldn't it make more sense to silo each macro in its own sql file?

Reply:
works for me!
pushing update soon.

Reply:
I just want names to not be confusing, `uniswap_fork` makes it seem like Uniswap is behind it with, similar to how `nft_base_ethereum.trades` seems to be related to the blockchain base although it isn't (even for forks imo). My suggestion would be to avoid using any already established names unless that name is directly linked/behind that data, that could also mean establishing some standardised way to name forks. Maybe something like `unifork`, `uniswapfork`, `dex_unifork` or whatever else doesn't contain the full `_uniswap_` name

Btw, @0xBoxer made a PR for all Uniswap v2 forks at ETHLisbon this weekend, this probably is the most scalable method as all that's needed is name mapping on top (which is part of the PR). https://github.com/duneanalytics/spellbook/pull/4730

Also curious about @0xRobin & @aalan3's thoughts here

Reply:
When it comes to `base` I agree that it can be confusing. The way to "tell" the difference is that `base` in the namespace (`nft_base....`) would be the chain, whereas in the table name (`....trades_base`) it refers to what we are doing here. Alternative could be maybe `core` that was suggested by someone? Another option could be to do some sort of `parent` or `main` naming. (btw Zora being a project and a chain also adds some more confusion to our taxonomy). 

Personally I actually associate `uniswap_fork` with non official uniswap things (pancakeswap etc), I think because usually in open source projects a fork means that someone broke out of the original project and created a separate org (but ofc not always). Curious if the uniswap contracts have a different name that could be used, like `seaport` vs `opensea`. Only thing I could find was `uniswap protocol Vx`.

Reply:
> Personally I actually associate uniswap_fork with non official uniswap things (pancakeswap etc),
good point. i can suggest a `_compatible` postfix instead of `_fork`

> When it comes to base I agree that it can be confusing
seems that we had a consesnsus in the other thread and decided to name these things as `_stg`
https://github.com/duneanalytics/spellbook/pull/4533/files/bd6ff597cd299203cfd03d498cc65b6783cfaeb2#diff-60520bb14954436eba8e725c491507a5165a3f6b8a2a0fcff85af99cf6f7cb1b

Reply:
>good point. i can suggest a _compatible postfix instead of _fork

I like `_compatible` because it works for both the forks and uniswap itself.

I'll look at the link, thanks!